nxproxy/configure.ac

```diff
@@ -0,0 +1,31 @@
+dnl ***************************************************************************
+dnl *** configure.ac for nxproxy                                         ***
+dnl ***************************************************************************
+
+m4_define([nxproxy_version], m4_esyscmd([tr -d '\n' < VERSION]))
+
+# Initialize Autoconf
+AC_PREREQ(2.60)
```

Comment:
We're raising the autoconf requirement from 2.13 to 2.60 here. Might be totally okay, but might also cause trouble on older distros (like The QVD or @uli42 use.)